app/src/main/java/com/infomaniak/drive/utils/DownloadOfflineFileManager.kt

```diff
@@ -226,10 +227,10 @@ class DownloadOfflineFileManager(
     companion object {
         private const val MAX_INTERVAL_BETWEEN_PROGRESS_UPDATE_MS = 1000L
 
-        fun observeBulkDownloadOffline(context: Context) = WorkManager.getInstance(context).getWorkInfosLiveData(
+        fun observeFailureDownloadWorkerOffline(context: Context) = WorkManager.getInstance(context).getWorkInfosLiveData(
```

Comment:
This function is named `observe…()`, but it doesn't observe anything.
It's just creating the liveData that is then observed in the calling method.

Can you rename it ?
Maybe something in the likes of `getFailureDownloadWorkerOffline()` ?